### ISSUE TITLE
delayed as resizing in apps

### ISSUE THREAD

**Issue by MosaIssei** [AvaloniaUI/Avalonia#9454]
When I resize in Windows and web applications, they resize with a delay

like this

![Untitled](https://user-images.githubusercontent.com/46990203/202259676-b313b80d-6e4d-4b78-b51b-e6cf131ed1be.png)

- OS: Windows 11
- Version 11 pew4

**Comment by lukaszossowski** [comment]
How long does it take to catch up? A short delay is quite common on Windows apps, even the ones from Microsoft.

**Comment by maxkatz6** [comment]
Yes, it's common. But from the discussion in the telegram chat, it seems to be slower than it was in the 0.10.18.
